src/main/resources/bidder-config/yahooAds.yaml

```diff
@@ -1,14 +1,16 @@
 adapters:
-  yahooAdvertising:
+  yahooAds:
```

Comment:
From issue 
https://github.com/prebid/prebid-server-java/issues/2417

We think that `all aliases should be enabled by default`

Please remove 'enabled: false' from all aliases

Reply:
It seems I can not simply remove the lines as empty aliases configs throw a null pointer error. I didn't investigate deeply to see the exact issue but I put some duplicate endpoint entries.